content/webapp/services/prismic/api.ts

```diff
@@ -0,0 +1,40 @@
+import * as prismic from 'prismic-client-beta';
+import { PrismicDocument } from '@prismicio/types';
+import { GetServerSidePropsContext } from 'next';
+import fetch from 'node-fetch';
+
+type Req = GetServerSidePropsContext['req'];
+
+const routes = [
+  {
+    type: 'seasons',
+    path: '/seasons/:uid',
+  },
+];
+const endpoint = prismic.getEndpoint('wellcomecollection');
+const client = prismic.createClient(endpoint, { routes, fetch });
+
+// We enforce have a req object as we should never be using the API on the client.
+// To fetch stuff via the client, use the ./api paths.
+const initialised = false;
+export function api(req: Req): prismic.Client {
+  if (!req) {
+    throw Error('Give me a req');
+  }
+  if (initialised) {
+    throw Error(
+      'Prismic has already been initialised, please use the previously initialised API reference'
```

Comment:
How often do we want to "initialise" an instance of the API (ie, I don't think it's once per application startup, right?) / can we say that in this message?

Reply:
I've adda comment saying we should do this once per request.

Reply:
I wonder if there's a way to enforce this... Say you initialise it with a request (or a request ID or something) and then if you try to use it with another request/ID it complains.

Reply:
Which is what we're doing, except the state is managed in this module. I could set `initialised = uuid()` or something, but a boolean feels good enough?

Reply:
Not sure that I understand, think we're talking at cross purposes - what stops us re-using an instance across multiple requests? When does `initialised` get reset to `false` on the next request? I can see that we enforce that only 1 instance is used for a given request, but I can't see how we enforce that it's used for exactly one request...

Reply:
@jamieparkinson I've gone for a more document / typed approach that actually doesn't throw, but should hopefully keep us on the straight and narrow as we develop.

What do you thin
k?

Reply:
I like this, nice